src/commands/view.py

```diff
@@ -9,6 +11,15 @@ def add_commands(appliance):
     def view():
         pass
 
+    @view.command(help='Lists the available groups')
+    def groups():
+        click.echo_via_pager("\n".join(groups_util.list()))
+
+    @view.command(help='Gives the nodes within a group')
```

Comment:
Only comment is shouldn't this be 'Lists' for consistency rather than 'GIves'? - other than that LGTM